### ISSUE TITLE
Voting Participation Rate not relative to eligible referenda

### ISSUE THREAD

**Issue by ggwpez** [opensquare-network/subsquare#4828]
The "Participation Rate" of a fellow seems to be calculated from the overall number of referenda, not the tracks that their are eligible to vote on.  

For example [here](https://collectives.subsquare.io/user/16a357f5Sxab3V2ne4emGQvqJaCLeYpTMx3TCjnQhmJQ71DX/votes):
<img width="578" alt="Screenshot 2024-10-08 at 12 03 24" src="https://github.com/user-attachments/assets/1dfdf7ee-87ec-47c0-b18a-d6d52f6e918f">

I would assume to have more than 40% though... the tracks that a rank 3 member can vote on are `1, 2, 3, 11, 21, 33`. So once we have such a mapping of rank to eligible tracks, it would hopefully be possible to show it relative to that.  
For example [here](https://github.com/polkadot-fellows/Evaluations/blob/main/0000-evidence-template.md#voting-record) in the rank evaluations, these statistics come in very handy.  
LMK if you need more examples of what ranks are eligible or some other info and whether its possible to add this feature 😄

**Comment by wliyongfeng** [comment]
Hmm..., you're right, but it's a little complex. I'll think how to handle this.

**Comment by wliyongfeng** [comment]
We'd like to hide this field for fellowship before the right data.

**Comment by ggwpez** [comment]
> Hmm..., you're right, but it's a little complex. I'll think how to handle this.

Yes I guess since you need to re-sync and evaluate at every block on which tracks the fellow could vote on?  
I dont see a simpler solution than that...

**Comment by kianenigma** [comment]
From which page can I navigate in to this? 

From fellowship > Members, when I clock on any user, I jump to /user.

**Comment by wliyongfeng** [comment]
@kianenigma Click the last button please. I agree it's not so convenient.

![Image](https://github.com/user-attachments/assets/24680734-8a0f-4af5-b850-651615a8b524)
